katdal/test/test_lazy_indexer.py

```diff
@@ -238,10 +267,28 @@ def test_stage1_multiple_boolean_indices(self):
         self._test_with(tuple([True] * d for d in self.data.shape))
         self._test_with(tuple([True, False] * (d // 2)
                               for d in self.data.shape))
+        self._test_with(np.s_[UNEVEN, 2 * UNEVEN, :24])
 
     def test_stage2_multiple_boolean_indices(self):
         stage1 = tuple([True] * d for d in self.data.shape)
         stage2 = tuple([True] * 4 + [False] * (d - 4) for d in self.data.shape)
         self._test_with(stage1, stage2)
         stage2 = tuple([True, False] * (d // 2) for d in self.data.shape)
         self._test_with(stage1, stage2)
+        stage1 = np.s_[UNEVEN, 2 * UNEVEN, :24]
+        stage2 = np.s_[:3, [1, 2, 3, 4, 6, 9], [8, 6, 4, 2, 0]]
+        self._test_with(stage1, stage2)
```

Comment:
This last check doesn't match the name of the test (multiple boolean indices).

Reply:
Fixed.